text-dream/public/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
what is the favicon? where from?

Reply:
Oh, I started this from a template I have and this favicon was from another project. I just forgot to remove it. We need to add a different one later.